src/main/java/com/redhat/devtools/lsp4ij/console/actions/ApplyLanguageServerSettingsAction.java

```diff
@@ -41,11 +59,82 @@ public void actionPerformed(@NotNull AnActionEvent e) {
+            // Only show the tooltip once per configuration change
+            if (modified && !hasSaveTipBalloonShown && showSaveTipOnConfigurationChange) {
+                hasSaveTipBalloonShown = true;
+                showBalloon(project);
+            }
+            // If configuration is returned to match the unmodified state, reset the tooltip to be shown again
+            if (!modified) {
+                hasSaveTipBalloonShown = false;
+            }
+        }
+
+        e.getPresentation().setEnabled(modified);
     }
 
     @Override
     public @NotNull ActionUpdateThread getActionUpdateThread() {
         return ActionUpdateThread.BGT;
     }
+
+    /**
+     * Create and show a balloon to draw attention to the save button when modifying
+     * configurations in the LSP console
+     * @param project passed on to the panel creation to create the button's action listener
+     */
+    private void showBalloon(@NotNull Project project) {
+        var jbPanel = createBalloonPanel(project);
+
+        BalloonBuilder builder = JBPopupFactory.getInstance()
+                .createBalloonBuilder(jbPanel)
+                .setFadeoutTime(1600) // How many ms the balloon is shown for
+                .setHideOnAction(false);
+
+        // Have an instance reference to hide the balloon with the button
+        this.saveTipBalloon = builder.createBalloon();
+
+        var components = languageServerView.getComponent().getComponents();
+        Optional<Component> actionToolbarComponent = Arrays.stream(components)
+                .filter(c -> ACTION_TOOLBAR_COMPONENT_NAME.equals(c.getName()))
+                .findFirst();
+
+        if (actionToolbarComponent.isEmpty()) {
+            return;
+        }
+        Component applyComponent = ((ActionToolbarImpl) actionToolbarComponent.get()).getComponent(0);
+
+        // Move the position by 1/2 of the component width, because the icon is not centered
+        Point point = new Point(applyComponent.getWidth()/2 ,0);
+        RelativePoint displayPoint = new RelativePoint(applyComponent, point);
+
+        // Use invokeLater to prevent the balloon from resizing when it is shown
+        ApplicationManager.getApplication().invokeLater(() -> this.saveTipBalloon.show(displayPoint, Balloon.Position.above));
+    }
+
+    /**
+     * Create the balloon panel that contains the save tip and disable button stacked on top of each other
+     * @param project necessary for updating the settings
+     * @return the created JBPanel
+     */
+    private @NotNull JBPanel<JBPanel> createBalloonPanel(Project project) {
+        var jbPanel = new JBPanel<>();
+        jbPanel.setLayout(new BoxLayout(jbPanel, BoxLayout.Y_AXIS));
+        JBLabel jbLabel = new JBLabel(LanguageServerBundle.message("action.lsp.detail.apply.balloon"));
+        JButton jButton = new JButton(LanguageServerBundle.message("action.lsp.detail.apply.balloon.disable"));

```

Comment:
To be consistent with IJ like 

![image](https://github.com/redhat-developer/lsp4ij/assets/1932211/f7cbdf15-1ab2-42bb-a589-3fc217758050)

It would be nice to use an hyperlink